desktop/mac/release-notes/index.md

```diff
@@ -29,9 +29,41 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
 > chip](https://desktop.docker.com/mac/main/arm64/Docker.dmg?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-mac-arm64){:
 > .button .primary-btn }
 
+### Security
+
+- Fixed CVE-2021-45449 that affects users currently on Docker Desktop version 4.3.0 or 4.3.1
```

Comment:
Should we link the CVE here? Or maybe the security docs page? cc @jerae-duffin

Reply:
I added the link to the CVE as we haven’t yet added a section for it to our docs page.
Obviously we can change that later.